### PR TITLE
RPRBLND-1865: Update Readme.md

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
Update Readme.md because it is outdated

### EFFECT OF CHANGE
Readme.md is up to date

### TECHNICAL STEPS
Added Build section
Fixed links
Removed outdated info
Added more explanation to debugging
Chenget sections order
Formatted text

### CODE REVIEW

**Review by bnagirniak — APPROVED** [review]
Looks good

File: README.md [hunk 1]
```diff
@@ -14,148 +14,175 @@ With Pixar's USD system emerging as a powerful tool for 3D graphics pipelines an
...
+
+For users who wish to install 3rd party render delegates (see above), they should be installed to the `libs/plugins/usd` directory in the addon folder similar to a regular USD installation.
 
 ## Usage
 ### Rendering
-At a simple level, this functions similar to any render addon to Blender, like Cycles or EEVEE which are included in Blender.  Simply select the render engine (in this case "USD Hydra") and render using the F12 key or starting a viewport render.  However, Hydra allows the added benefit of selecting a "Render Delegate".  The default is using Radeon ProRender, AMD's cross device GPU path tracer.
+At a simple level, this functions similar to any render addon to Blender, like Cycles or EEVEE which are included in Blender. Simply select the render engine (in this case "USD Hydra") and render using the `F12` key or starting a viewport render. However, Hydra allows the added benefit of selecting a "Render Delegate". The default is using Radeon ProRender, AMD's cross device GPU path tracer.
```
Comment (DagerD) [inline README.md:41]: Radeon ProRender is ray tracing, not path tracer

File: README.md [hunk 2]
```diff
@@ -14,148 +14,175 @@ With Pixar's USD system emerging as a powerful tool for 3D graphics pipelines an
...
-
-Also, make more meaningful commits(one commit per feature) the easy way: 
-
-`git merge <branch> --squash` # this will create a single change set from multiple commits coming from <branch>
+## Contributing
+### Build Requirements
+- [Blender 2.93+](https://www.blender.org/download/)
+- [Python 3.9 x64](https://www.python.org/ftp/python/3.9.7/python-3.9.7-amd64.exe) _(Blender 2.93+ uses 3.9)_
+  - numpy - `pip install numpy`
+  - requests - `pip install requests`
```
Comment (DagerD) [inline README.md:65]: pip install PyOpenGL
pip install PySide2

File: README.md [hunk 3]
```diff
@@ -14,148 +14,175 @@ With Pixar's USD system emerging as a powerful tool for 3D graphics pipelines an
...
+```commandline
+> git clone https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon
+> cd BlenderUSDHydraAddon
+> git submodule update --init --recursive
+> python tools/build.py -all -bin-dir bin
+```
+
+#### Build tool
+You can build project using `tools/build.py` with different flag combinations. It allows you to create a folder with binaries and copy all the necessary files for development to `/libs` folder. Also `tools/build.py` provides a verity of ways to make a project builds:
+- `-all` - builds all binaries
```
Comment (DagerD) [inline README.md:127]: and copies libs to folder (includes -libs flag)
and generates zip (includes -addon flag)

Maybe should add that -all equals to -usd -hdrpr -mx -libs -mx-classes -addon

**Review by DagerD — CHANGES_REQUESTED** [review]
Commented in text

File: README.md (hunk 3 above)
Comment (VascoPi) [reply README.md]: Thanks, added more description.

File: README.md (hunk 2 above)
Comment (VascoPi) [reply README.md:65]: Thanks, added

**Review by DagerD — APPROVED** [review]
Ok

**Pull request merged**